ext_authz/Dockerfile-opa

```
@@ -1,3 +1,3 @@
-FROM openpolicyagent/opa:0.70.0-istio@sha256:4d9b0636e04635fcd993141346ddbb5b8b96bd6f8c511bdc0b65a5d0e4f740b2
+FROM openpolicyagent/opa:0.70.0-istio-static@sha256:44a21c67041d086d11e2404df22c146993ca691ace24e46f18d9c88b2f41e26b
```

```suggestion
FROM openpolicyagent/opa:0.70.0-istio-static@sha256:808342677946849bd359e519666a0b5097885961d2a8f6482e0c99b598e2a996
```
best use this sha as its cross-arch compatible